src/cmd_line/subparsers/file.ts

```diff
@@ -9,7 +9,7 @@ export function parseEditFileCommandArgs(args: string): node.FileCommand {
     }
 
     var scanner = new Scanner(args);
-    let name = scanner.nextWord();
+    var name = scanner.nextWord();
```

Comment:
oh your suggestion is using `let` whenever possible.

so previously when you said `use var, not let` in ` parseEditFileInNewWindowCommandArgs` is kind of typo? :)